ocs_ci/ocs/resources/packagemanifest.py

```diff
@@ -135,7 +135,10 @@ def get_channels(self):
         """
         self.check_name_is_specified()
         try:
-            return self.data["status"]["channels"]
+            if isinstance(self.data, list):
+                return self.data[0]["status"]["channels"]
```

Comment:
Are we sure, that the expected one is always the first one?

Reply:
yes, i have tested it with few GA version, and the used channel was all the time the first one in the list.